autoload/ale/erlang.vim

```diff
@@ -0,0 +1,9 @@
+" Author: Kohei YOSHIDA - https://github.com/yosida95
+" Description: Helper functions for Erlang tools
+
+call ale#Set('erlang_rebar_executable', 'rebar3')
+call ale#Set('erlang_rebar_use_global', get(g:, 'ale_use_global_executables', 0))
+
+function! ale#erlang#GetRebarExecutable(buffer) abort
```

Comment:
There's a runtime cost in loading this file, and we only have this one function and call it from one place. Let's just move this into the fixer file instead, to avoid this runtime cost.

Reply:
Done